src/syft/lib/action/__init__.py

```diff
@@ -0,0 +1,60 @@
+# stdlib
+from typing import Any as TypeAny
+from typing import List as TypeList
+from typing import Tuple as TypeTuple
+
+# syft absolute
+from syft.ast import add_classes
+from syft.ast import add_methods
+from syft.ast import add_modules
+from syft.ast.globals import Globals
+from syft.core.node.common.action.get_object_action import GetObjectAction
+from syft.core.node.common.plan.plan import Plan
+
+# syft relative
+from . import plan_wrapper  # noqa: 401
+
+
+def create_plan_ast(client: TypeAny = None) -> Globals:
+    ast = Globals(client)
+
+    modules = [
+        "syft",
+        "syft.core",
+        "syft.core.node",
+        "syft.core.node.common",
+        "syft.core.node.common.action",
+        "syft.core.node.common.action.get_object_action",
+        "syft.core.node.common.plan",
+        "syft.core.node.common.plan.plan",
+    ]
+
+    classes: TypeList[TypeTuple[str, str, TypeAny]] = [
+        (
+            "syft.core.node.common.action.get_object_action.GetObjectAction",
+            "syft.core.node.common.action.get_object_action.GetObjectAction",
+            GetObjectAction,
+        ),
+        (
+            "syft.core.node.common.plan.plan.Plan",
```

Comment:
Let's create a convenience reference for "Plan" right off of syft...aka

from syft import Plan

:)

Reply:
Tbh, I am not sure how to implement this

Reply:
just import `Plan` into the main `Syft` init:
```
# Convenience Constructors
from syft.core.node.vm.vm import VirtualMachine  # noqa: F401
from syft.core.node.vm.vm import VirtualMachineClient  # noqa: F401

from syft.core.node.common.plan.plan import Plan  # noqa: F401
```

That will make it available at the top level.

Also, could we maybe move all of Plans into `core` the same level as `pointer`?
<img width="207" alt="Screen Shot 2021-02-23 at 4 49 19 pm" src="https://user-images.githubusercontent.com/2882739/108810538-13883e80-75f7-11eb-9f82-01ac50d73e4a.png">

Reply:
Ah oke, I misunderstood